packages/lab/src/app-header/AppHeader.css

```diff
@@ -1,6 +1,15 @@
+.uitk-density-high,
+.uitk-density-medium,
+.uitk-density-low {
+  --appHeader-height: var(--uitkAppHeader-height, calc(var(--uitk-size-base) + var(--uitk-size-unit) * 2));
+}
+
+.uitk-density-high {
```

Comment:
Any reason you are duplicating high here?

Reply:
sorry well spotted! this single one should have been touch i assume, will check and change